blueocean-dashboard/src/main/less/core.less

```diff
@@ -64,7 +64,7 @@
     margin-top: 10px;
 
     th {
-        width: 75px;
+        width: 100px;
```

Comment:
Should every column be pushed to 100px, or just the one that needs it? Could be that all the other columns are already getting explicit widths down below... although if we want to size the "actions" column explicitly, it might be good to slap a `className` on that one.